source/MRViewer/MRSurfaceContoursWidget.h

```diff
@@ -144,6 +144,14 @@ class MRVIEWER_CLASS SurfaceContoursWidget : public MultiListener<
     // data storage
     SurfaceContours pickedPoints_;
 
+    // connection storage
+    struct SurfaceConnectionHolder
+    {
+        boost::signals2::connection onMeshChanged;
+        boost::signals2::connection onPointsChanged;
+    };
+    std::unordered_map<std::shared_ptr<VisualObject>, SurfaceConnectionHolder> surfaceConnectionHolders_;
```

Comment:
does it really need to be map? callback knows about obj from lambda capture, mb just vector of connections is enough?